src/dom_components/view/ComponentTextView.ts

```diff
@@ -182,13 +182,13 @@ export default class ComponentTextView extends ComponentView<ComponentText> {
       const offset = range.startOffset;
       const textModel = getComponentModel(textNode);
       const newCmps: (ComponentDefinition | Component)[] = [];
-
+      let data = textNode.textContent ?? '';
       if (textModel && textModel.is?.('textnode')) {
         const cmps = textModel.collection;
         cmps.forEach(cmp => {
           if (cmp === textModel) {
             const type = 'textnode';
-            const cnt = cmp.content;
+            const cnt = (data = '' ? cmp.content : data);
```

Comment:
important: use equality operator (`==`) or strict equality operator (`===`)

suggestion: `!data ? cmp.content : data;` (grouping operator `(...)` is not necessary here)

Reply:
done